azure-pipelines-harrastukset-release.yml

```diff
@@ -46,5 +46,22 @@ extends:
   # Django example: azure-pipelines-PROJECTNAME-api-release.yml
   # Drupal example: azure-pipelines-drupal-release.yml
   template: azure-pipelines-harrastukset-release.yml@harrastukset-pipelines
-  #parameters:
-    #imagetag: ${{ parameters.imagetag }}
+  parameters:
+    # Application build arguments and config map values as key value pairs.
+    # Does not contain all buildArguments, the rest located in harrastukset-pipelines/harrastukset-variables-staging.yml and harrastukset-variables-common.yml
+    # The values here will override the values defined in the harrastukset-pipelines repository
+    buildArgs:
+      NEXTJS_DISABLE_SENTRY: 1
+      NEXT_PUBLIC_ALLOW_UNAUTHORIZED_REQUESTS: 0
+      NEXT_PUBLIC_CMS_ORIGIN: https://harrastus.content.api.hel.fi
+      NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT: https://api.hel.fi/linkedevents/v1/event
+      NEXT_PUBLIC_DEBUG: 0
+      NEXT_PUBLIC_MATOMO_ENABLED: 1
+    configMap: # pod environment variables
+      NEXT_PUBLIC_ALLOW_UNAUTHORIZED_REQUESTS: 0
+      NEXT_PUBLIC_CMS_ORIGIN: https://harrastus.content.api.hel.fi
+      NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT: https://api.hel.fi/linkedevents/v1/event
+      NEXT_PUBLIC_DEBUG: 0
+      NEXT_PUBLIC_MATOMO_ENABLED: 1
```

Comment:
Are the parameters really needed in both mappings? For example, if an URL is used build time, I can't see any reason why it would be different in the running environment -- Why would I create some static HTML files out of CMS content in build time, but use some other CMS in a runtime? I could understand that some flags like `debugging` could be switched off in build time, but set on in runtime, or viceversa, but it's not the same with the URLs.

Reply:
However, the idea that the variables are now here in the monorepo, is a really good thing! 👏

Reply:
This is done as it is configured now. The ticket HH-299 is for dividing variables into build and runtime.